sys/kern/uipc_shm.c

```diff
@@ -1430,16 +1432,18 @@ kern_shm_rename(struct thread *td, const char * __capability path_from_p,
 }
 
 static int
-shm_mmap_large(struct shmfd *shmfd, vm_map_t map, vm_offset_t *addr,
+shm_mmap_large(struct shmfd *shmfd, vm_map_t map, vm_pointer_t *addr,
     vm_offset_t maxaddr,
     vm_size_t size, vm_prot_t prot, vm_prot_t max_prot, int flags,
     vm_ooffset_t foff, struct thread *td)
 {
 	struct vmspace *vms;
-	vm_map_entry_t next_entry, prev_entry;
-	vm_offset_t align, mask;
+	vm_offset_t align, mask, vaddr;
 	int docow, error, rv, try;
 	bool curmap;
+	vm_pointer_t start;
+	vm_offset_t reservation_id;
```

Comment:
style(9) pushes for sorting variables by size. I'm not a stickler for it, but noticed it in this case.
```suggestion
	vm_pointer_t start;
	vm_offset_t align, mask, reservation_id, vaddr;
	int docow, error, rv, try;
	bool curmap;
```